content/add_displayname_to_json_files.py

```diff
@@ -0,0 +1,25 @@
+import os
+import json
```

Comment:
These are some of the script I wrote to help add fields to .md and .json files, as well as tell when files had different filename fields an actual filenames. 

Are these worth keeping and if so where should they live?

Reply:
I'm neutral on keeping the scripts, but sure, might as well. You could put them in a `content/scripts' directory